_search-plugins/cache-plugins/index.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: default
+title: Cache types
```

Comment:
This folder is named `cache-plugins`, I m guessing we will have to rename that as well?

Reply:
Thanks @kolchfa-aws for renaming this. "Caching" looks good to me.